ProjectSettings/TagManager.asset

```diff
@@ -38,6 +38,12 @@ TagManager:
   - 
   - 
   m_SortingLayers:
+  - name: Background
+    uniqueID: 847899343
+    locked: 0
   - name: Default
     uniqueID: 0
     locked: 0
+  - name: Player
```

Comment:
It seems like the added sorting layers are unused.

Reply:
Gotcha; I deleted the "Player" layer and used the "Background" layer for the background.